pyproject.toml

```diff
@@ -9,29 +9,36 @@ select = ["ALL"]
 extend-ignore = [
     "BLE",
```

Comment:
what is all this stuff ?

Reply:
ruff is a Python linter that I discovered recently and am testing at the moment.  It is a replacement for flake8 but implemented in Rust and thus much faster.  It's rule set also includes those of a lot of flake8-plugins (see [list of supported rules](https://beta.ruff.rs/docs/rules/).  Since I wasn't sure which ones I want to use and which not, I simply enabled all and then and then selectively disable (through `extend-ignore`) those that I don't like.